### ISSUE TITLE
Execute multi-line select sql failed

### ISSUE THREAD

**Issue by rj-hwang** [r2dbc/r2dbc-postgresql#80]
Using 1.0.0.M6 execute bellow multi-line select sql failed:

```
select id from T
  where id in ($1, $2)
  order by id asc
```

r2dbc-postgres error:

```
Suppressed: java.lang.UnsupportedOperationException: Binding parameters is not supported for the statement ...
```

r2dbc-h2 error:

```
Suppressed: org.h2.message.DbException: Method is not allowed for a query. Use execute or executeQuery instead of executeUpdate [90001-197]
```

But execute success if change to a single line sql such as 'select id from T where id in ($1, $2) order by id asc'.

**Comment by mp911de** [comment]
Have you retested the scenario with 1.0.0.M7?

**Comment by rj-hwang** [comment]
The problem fixed after upgrade to 1.0.0.M7.

**Comment by mp911de** [comment]
Thanks for retrying and the update.
